lib/v2/screens/transfer/receive/receive_enter_data/interactor/mappers/create_invoice_result_mapper.dart

```diff
@@ -8,11 +10,10 @@ class CreateInvoiceResultMapper extends StateMapper {
   ReceiveEnterDataState mapResultToState(ReceiveEnterDataState currentState, Result result) {
     if (result.isError) {
       print('Error invoice hash not retrieved');
-      return currentState.copyWith(
-        pageState: PageState.success,
-        pageCommand: ShowTransactionFail(),
-      );
+      return currentState.copyWith(pageState: PageState.success, pageCommand: ShowTransactionFail());
     } else {
+      eventBus.fire(const OnNewTransactionEventBus());
+
```

Comment:
Not sure if i can say here is where a recibe invoice was success.

Reply:
No there is no way to know a receive has worked, no need to do this here, since this is just the receive result screen. This should not be here, can remove.
For now user has to refresh screen manually to see incoming receive
In the future we should handle push notification that a transaction occurred, and refresh on that.

Reply:
Please remove this code

Reply:
go it

Reply:
removed